src/css/profile/mobile/common/button.less

```diff
@@ -405,19 +432,44 @@ a.ui-btn {
 				opacity: 1;
 			}
 			&-top {
-				padding: 64 * @px_base 0 0 0;
+				line-height: normal;
+				height: 60 * @px_base;
+				display: flex;
+				align-items: center;
+				flex-direction: column-reverse;
+				justify-content: center;
+				padding: 0;
+				color: var(--appbar-subtitle-color);
+
 				&::after {
-					top: 0;
+					top: auto;
+					left: auto;
+					border-radius: 0;
+					width: 24 * @px_base;
+					height: 24 * @px_base;
+					mask-size: 24 * @px_base;
+					position: relative;
+					padding-bottom: 1 * @px_base;
+					transform: none;
+					background-color: var(--bottom-button-icon-color);
 				}
 				&::before {
-					top: 0;
-					transform: translateX(-50%);
+					top: auto;
+					left: auto;
+					border-radius: 12 * @px_base;
+					transform: none;
+					width: 100%;
+					height: 60 * @px_base - 4 * @px_base;
```

Comment:
why there is -4.
Ripple effect is not covering whole button in such case?

Reply:
You have to check guideline

Reply:
ok